git_app/cli.py

```diff
@@ -91,18 +91,18 @@ def cli(
     max_count,
     trace_preview_length,
     service_dict,
-    commit_hash,
+    ref,
 ):
     """Run base command for varangian git forge application."""
-    lib.run(
+    lib.create_issue_with_comments(
         repo=repo,
         predictions_file=predictions_file,
         trace_directory=trace_directory,
         namespace=namespace,
-        max_count=max_count,
+        # max_count=max_count,
```

Comment:
Is commenting a line in function arguments allowed?
will this not cause any issu?

Reply:
I will remove this line

Reply:
we might have to change it in other places in the pr , the other function are also defined with comment inside the argument, please bring them outside of the function definition.